nservicebus/hosting/critical-errors.md

```diff
@@ -31,11 +31,13 @@ Define a custom handler using the following code.
 
 snippet: DefiningCustomHostErrorHandlingAction
 
-### Terminating the process
+### Stopping the endpoint
```

Comment:
Maybe we can extend that with:

Windows Service -> FailFast
Linux Daemon -> Failfast
Dockter container -> Failfast

But all with recovery configured.

Reply:
this might be something that would make more sense in dedicated documentation for the common hosting options, e.g. separate pages for windows service and linux service hosting and configuration. E.g. different cloud service might also require different behavior.